Formula/osv-scanner.rb

```diff
@@ -0,0 +1,28 @@
+class OsvScanner < Formula
+  desc "Vulnerability scanner which uses the OSV database"
+  homepage "https://github.com/google/osv-scanner"
+  url "https://github.com/google/osv-scanner.git",
+      tag:      "v1.0.0",
+      revision: "c6ed0fabdbe8ad75465bd7eab81ba904a9c89f42"
+  license "Apache-2.0"
+  head "https://github.com/google/osv-scanner.git", branch: "main"
+
+  depends_on "go" => [:build]
+
+  def install
+    system "go", "build", *std_go_args, "-o", bin/"osv-scanner", "./cmd/osv-scanner/"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "./cmd/osv-scanner/"
```
This is a standard argument